### PR TITLE
docs(filter): add examples of unregister

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

#834

### CODE REVIEW

File: source/api/filter.md [hunk 1]
```diff
@@ -57,6 +57,28 @@ hexo.execFilterSync(type, data, options);
 hexo.extend.filter.unregister(type, filter);
 ```
 
+**Example**
+
+``` js
+// Unregister a filter which is registered with named function
+
+const filter = (data) => {
```
Comment (curbengh) [inline source/api/filter.md]: suggest to named it something else (`imgFn` perhaps?) other than filter, otherwise may confuse with `hexo.extend.`**filter**

Comment (SukkaW) [reply source/api/filter.md]: Maybe I will rename it to filterFn.

Comment (curbengh) [reply source/api/filter.md]: I initially thought of that as well, but there's still a `filter` in it. Ultimately, I think it should be fine, as long it's not `filter`.

Comment (SukkaW) [reply source/api/filter.md]: Updated.

**Pull request merged**